### PR TITLE
Add free.fr SMS notification provider

### PR DESCRIPTION
**Pull request by CL0Pinette**
⚠️⚠️⚠️ Since we do not accept all types of pull requests and do not want to waste your time. Please be sure that you have read pull request rules:
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma

Tick the checkbox if you understand [x]: 
- [x] I have read and understand the pull request rules.

# Description

The french ISP https://mobile.free.fr provides an API to send SMS over a web requests to your own phone number.
I just set up uptime-kuma on my server and realized that it could be nice if I could receive updates on my phone when a service stops. 

## Type of change

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and tested it
- [x] I have commented my code, particularly in hard-to-understand areas
  (including JSDoc for methods)
- [x] My changes generate no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

## Screenshots (if any)

Please do not use any external image service. Instead, just paste in or drag and drop the image here, and it will be uploaded automatically.

### CODE REVIEW

**Comment by louislam** [comment]
Nice, this type of pr will be accepted.

**Comment by CL0Pinette** [comment]
Great ! I will update the PR in the following days with the new modifications 👍

File: server/notification-providers/freemobile.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+const axios = require("axios");
+
+class FreeMobile extends NotificationProvider {
+
+    name = "FreeMobile";
+
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        let okMsg = "Sent Successfully.";
+        try {
+            await axios.post(`https://smsapi.free-mobile.fr/sendmsg?msg=${encodeURI(msg.replace("🔴", "⛔️"))}`, {
```
Comment (louislam) [inline server/notification-providers/freemobile.js:11]:
```suggestion
            await axios.post(`https://smsapi.free-mobile.fr/sendmsg?msg=${encodeURIComponent(msg.replace("🔴", "⛔️"))}`, {
```

Comment (louislam) [reply server/notification-providers/freemobile.js:11]: Usually, it should be `encodeURIComponent`, but I don't know how they handled this, you can test these characters.

image

Comment (CL0Pinette) [reply server/notification-providers/freemobile.js]: I tested all these chars and all of them are working properly, should be good now

**Comment by MichelBaie** [comment]
Merci beaucoup !! 👍👍

**Pull request merged**